source/sync/error-handling/errors.txt

```diff
@@ -148,19 +148,36 @@ The following errors may occur when your {+app+} uses :ref:`Flexible Sync
        The client attempts to reconnect until this process is complete. Then,
        this error resolves and Sync begins working normally.
     
-   * - ErrorWriteNotAllowed
-     - This error occurs when a client attempts a write that is disallowed 
-       by permissions, or if the write modifies an object outside of the 
-       subscription query. 
-       
-       Consider an example where the client application opens a subscription 
-       on a ``Places`` object type with the query ``name = "NY"``. If the 
-       client then tries to insert a ``Places`` object with a field 
-       ``name: "CT"``, the write triggers an ``ErrorWriteNotAllowed`` because 
-       the object is not within the subscription query.
-
-       This error triggers a client reset. To recover from this error, 
-       perform a :ref:`client reset <client-resets>`.
+   * - ErrorCompensatingWrite
+     - This non-fatal error occurs when a client attempts an "illegal" write.
+       The following are considered illegal writes:
+
+       - Creating an object before opening a subscription
+       - Creating an object that would be outside of the client's query view
+       - Modifying an object that is outside of the client's query view
```

Comment:
can we clarify that "query view" includes both the client's subscriptions, as well as their read permissions?